reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -353,8 +358,20 @@ abstract class TableReporter : Reporter() {
             extraColumns.map { it.toString() }
         }.orEmpty()
 
-        return generateReport(TabularScanRecord(ortResult.repository.vcsProcessed, errorSummaryTable, summaryTable,
-                projectTables, metadata, extraColumns), outputDir)
+        val evaluatorErrors = ortResult.evaluator?.errors
```

Comment:
We *could* inline this now. Approving nonetheless.

Reply:
Done.